gradle/libs.versions.toml

```
@@ -1,5 +1,5 @@
 [versions]
-opendcs = "7.5.1-RC17"
+opendcs = "7.5.1-RC18"
 
 servlet-api = "6.1.0" #Updating this further will require a change to the jakarta namespace.
```

```suggestion
servlet-api = "6.1.0"
```

we um, just did that.